import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        TaskExecutor a = new TaskExecutor();
        a.Carregamento(4,100);
        a.Processamento(16 );
//        Acesso b = new Acesso(4);
//        System.out.println(b.read());
//        System.out.println(b.write(10));
//        System.out.println(b.read());
//        System.out.println(b.write(10));
//        System.out.println(b.write(10));
//        System.out.println(b.write(10));
//        System.out.println(b.read());
    }

}
